### PR TITLE
[4] Cleanup of switch statements to remove code smell & adhere to code style guidelines

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

Cleanup of switch statements to remove code smell & adhere to code style guidelines - mostly reported by inspection in phpStorm

Removes spaces before :

Removes redundant cases that are already caught by default, you don't need case statements on the same block as the default.

default commonly found as last block, not first, refactored one place to make it last

### Testing Instructions

Code review

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on ecb33bc7925f4bf9f7ea859ad477edfc38fee3b5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31058">issues.joomla.org/tracker/joomla-cms/31058</a>.</sub>

File: administrator/components/com_installer/src/Model/InstallerModel.php [hunk 1]
```diff
@@ -182,8 +182,7 @@ protected function translate(&$items)
 				case 'component':
 					$extension = $item->element;
 					$source = JPATH_ADMINISTRATOR . '/components/' . $extension;
-						$lang->load("$extension.sys", JPATH_ADMINISTRATOR)
-					||	$lang->load("$extension.sys", $source);
+					$lang->load("$extension.sys", JPATH_ADMINISTRATOR) ||	$lang->load("$extension.sys", $source);
```
Comment (SharkyKZ) [inline administrator/components/com_installer/src/Model/InstallerModel.php]:
```suggestion
					$lang->load("$extension.sys", JPATH_ADMINISTRATOR) || $lang->load("$extension.sys", $source);
```

File: administrator/components/com_installer/src/Model/InstallerModel.php [hunk 2]
```diff
@@ -203,25 +202,22 @@ protected function translate(&$items)
...
-						$lang->load("$extension.sys", $path)
-					||	$lang->load("$extension.sys", $source);
+					$lang->load("$extension.sys", $path) || $lang->load("$extension.sys", $source);
 				break;
 				case 'plugin':
 					$extension = 'plg_' . $item->folder . '_' . $item->element;
 					$source = JPATH_PLUGINS . '/' . $item->folder . '/' . $item->element;
-						$lang->load("$extension.sys", JPATH_ADMINISTRATOR)
-					||	$lang->load("$extension.sys", $source);
+					$lang->load("$extension.sys", JPATH_ADMINISTRATOR) ||	$lang->load("$extension.sys", $source);
```
Comment (SharkyKZ) [inline administrator/components/com_installer/src/Model/InstallerModel.php]:
```suggestion
					$lang->load("$extension.sys", JPATH_ADMINISTRATOR) || $lang->load("$extension.sys", $source);
```

**Comment by PhilETaylor** [comment]
Committed suggestions

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 1ab132431ac44897d3af9fe5919c9cfa23591e8f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31058">issues.joomla.org/tracker/joomla-cms/31058</a>.</sub>

**Comment by PhilETaylor** [comment]
well I screwed this PR somehow :(

**Comment by PhilETaylor** [comment]
Force pushing to fix rebase issue.. one moment

**Comment by PhilETaylor** [comment]
ok merge complete - no conflicts this time - ready for testing again

**Review by wilsonge — APPROVED** [review]

**Review by HLeithner — APPROVED** [review]
Code Review

**Pull request merged**